server/api/list/[_id].get.ts

```
@@ -1,6 +1,18 @@
+
+    if (!list) {
+      return {
+        statusCode: 404,
+        message: 'List not found',
+      }
+    }
+    const listTodos = await TodoSchema.find({ listId: list._id }).sort({ order: 1 })
+    console.log('list', list)
+    console.log('listTodos', listTodos)
+    return list
+    return { list, listTodos }
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)